### PR TITLE
Add admin stats

### PR DESCRIPTION
**Pull request by defsprite**
This adds support for "Admin Stats" part of the GitHub Enterprise API, as per https://developer.github.com/v3/enterprise-admin/admin_stats. It should implement part of #179 

At this point, only getting "all" stats is supported, thought it should be straightforward to add  others at some point. 

I'm really not sure about the naming here, the api prefixes for the GHE Admin API are all over the place with `/admin`, `/staff`, `/setup` and `/enterprise`. I see three options here:

- Stick to the pattern (In that case I'd need to move this implementation to `EnterpriseService`)
- Collect all of them in the existing `AdminService` 
- Collect all of them in a newly created `EnterpriseAdminService` (migration & deprecation fun)

### CODE REVIEW

**Comment by defsprite** [comment]
I signed it!

File: github/admin_stats.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+type RepoStats struct {
+	TotalRepos  *int `json:"total_repos,omitempty"`
+	RootRepos   *int `json:"root_repos,omitempty"`
+	ForkRepos   *int `json:"fork_repos,omitempty"`
+	OrgRepos    *int `json:"org_repos,omitempty"`
+	TotalPushes *int `json:"total_pushes,omitempty"`
+	TotalWikis  *int `json:"total_wikis,omitempty"`
+}
+
+func (s *AdminService) GetAdminStats(ctx context.Context) (*AdminStats, *Response, error) {
```
Comment (e-beach) [inline github/admin_stats.go]: I anticipate the maintainers will want you to add doc-sentences for this function and the structs, as in https://github.com/google/go-github/blob/61bf8a6a2009b7fb88173ed454c77fb8013b8852/github/licenses.go#L61-L64.

**Comment by shurcooL** [comment]
Thanks for the PR @defsprite.

As @e-beach mentioned, we'll definitely want the exported top-level identifiers to have documentation. See the other .go files for precedent and preferred format.

> I'm really not sure about the naming here, the api prefixes for the GHE Admin API are all over the place with `/admin`, `/staff`, `/setup` and `/enterprise`. I see three options here:
>
> - Stick to the pattern (In that case I'd need to move this implementation to `EnterpriseService`)
> - Collect all of them in the existing `AdminService` 
> - Collect all of them in a newly created `EnterpriseAdminService` (migration & deprecation fun)

Good question.

I think in this case there's an easy answer. Since we already have an existing `AdminService`, this feels like it would fit nicely there.

If there was no equivalent non-GHE service for a GHE endpoint, we might want to create an `EnterpriseService` and put all the stragglers there. I definitely don't think we ever need to go down the `EnterpriseAdminService`, there's just not that many GHE-only endpoints to justify it.

Thanks!

**Comment by shurcooL** [comment]
Thanks, looks good.

This is relatively minor, but would be nice to fix: many of the sentences of the documentations do not end with a period. They should, to be consistent with the rest of our documentation. Our goal is to use correct grammar, spelling, and have complete sentences with punctuation.

File: github/admin_stats_test.go [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+	})
+
+	stats, _, err := client.Admin.GetAdminStats(context.Background())
+	if err != nil {
+		t.Errorf("AdminService.GetAdminStats returned error: %v", err)
+	}
+
+	want := &AdminStats{
+		Repos: &RepoStats{
+			Int(212),
```
Comment (gmlewis) [inline github/admin_stats_test.go]: I'm not sure we need to test the value of every field, indeed we do not do elsewhere in this package if I recall correctly, as it becomes more challenging to maintain as fields are added.

However, I'm not adverse to it either if you explicitly use the field names here in the test and not rely solely on ordering. That way, a new field can more easily be added, or even if it is not, the unit test would not suddenly break.

Comment (defsprite) [reply github/admin_stats_test.go]: Absolutely, relying on order is super brittle!

**Comment by defsprite** [comment]
All done now. 

Please re-check @gmlewis

**Pull request merged**